automation/check-patch.e2e.setup.sh

```diff
@@ -1,27 +1,9 @@
-# Prepare environment for kubernetes-nmstate end to end testing. This includes temporary Go paths and binaries.
+# Prepare environment for kubernetes-nmstate end to end testing.
 #
-# source automation/check-patch.e2e.setup.sh
-# cd ${TMP_PROJECT_PATH}
 
-echo 'Setup Go paths'
-export GOROOT=/tmp/kubernetes-nmstate/go/root
```

Comment:
With this the script would kill my Go environment if executed locally. On top of that, it would not work in an environment where Go is not set up yet. The point of these scripts was to allow it to run anywhere without dependencies.

Reply:
going back to them